recipes/aws-c-mqtt/all/conanfile.py

```diff
@@ -0,0 +1,82 @@
+from conans import ConanFile, CMake, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+class AwsCMQTT(ConanFile):
+    name = "aws-c-mqtt"
+    description = "C99 implementation of the MQTT 3.1.1 specification."
+    topics = ("conan", "aws", "amazon", "cloud", )
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/awslabs/aws-c-mqtt"
+    license = "Apache-2.0",
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.cppstd
+        del self.settings.compiler.libcxx
+
+    def requirements(self):
+        self.requires("aws-c-common/0.6.7")
+        self.requires("aws-c-cal/0.5.11")
+        self.requires("aws-c-io/0.10.5")
+        self.requires("aws-c-http/0.6.5")
```

Comment:
There is an option for websocket: MQTT_WITH_WEBSOCKETS It could be added in the future. It seems like only depends on aws-c-http